uniffi_bindgen/src/interface/function.rs

```diff
@@ -381,20 +375,17 @@ mod test {
         let func2 = ci.get_function_definition("rich").unwrap();
         assert_eq!(func2.name(), "rich");
         assert_eq!(
-            func2.return_type().unwrap().canonical_name(),
```

Comment:
So yeah, when I moved canonical_name into the Ruby bindings, I didn't notice that function *was* used in tests, so the version in the Ruby bindings were actually being used here. I think these test changes are improvements though - we are now checking the *actual* type.